pkg/controller.v2/controller.go

```diff
@@ -391,9 +391,18 @@ func (tc *TFJobController) reconcileTFJobs(tfjob *tfv1alpha2.TFJob) error {
 		return err
 	}
 
+	rstatus := make(map[string]v1.PodPhase)
+	psReplicas := 0
+	workerReplicas := 0
 	// Diff current active pods/services with replicas.
 	for rtype, spec := range tfjob.Spec.TFReplicaSpecs {
-		err = tc.reconcilePods(tfjob, pods, rtype, spec)
+		if rtype == tfv1alpha2.TFReplicaTypePS {
+			psReplicas = int(*spec.Replicas)
+		}
+		if rtype == tfv1alpha2.TFReplicaTypeWorker {
+			workerReplicas = int(*spec.Replicas)
+		}
+		err = tc.reconcilePods(tfjob, pods, rtype, spec, rstatus)
```

Comment:
I prefer to return the map instead of pass it as a argument, since we do not use it except here.

Reply:
And, could you please explain the name `rstatus`? Do you mean `replicaStatus`?

Reply:
ok,I will do it